docs/technical_specs/core/util/unit_costing.rst

```diff
@@ -28,17 +28,24 @@ In addition, the first time `get_costing` is called for a unit operation within
 on the flowsheet object (i.e. `flowsheet.unit.costing`) in order to hold any global parameters relating to costing. The most 
 common of these paramters is the cost normalization parameter based on the year selected by the user.
 
+An initialize method for costing components has been developed. This method is called by the unit.initialize() or it can be imported by the user to initialize costing blocks.
```

Comment:
This is not quite correct, at least from the developer's stand point. I would reword this to be something like:

The unit costing module also contains an `initialize` method which can be used to estimate good initial guesses for costing variables based on the current stat of the associated unit model. This method can be called directly from the `unit_costing` module to initialize a specific costing block, or can be incorporated into a unit model initialization procedure. This method has been incorporated into the `initialize` method of all the models in the core unit model library,